community/get-involved.mdx

```diff
@@ -0,0 +1,108 @@
+- If there is any bug, please share steps to reproduce that bug.
+- If there is any issue in running Novu on a local machine, please share system details like operating system, RAM size, npm version and node version.
+- If there is any issue in our backend SDKs, please mention the SDK version and relevant details.
+- If looking for self-hosting support, please share `Novu Version` and remote server details. Our latest version is `0.21.0`. Please be patient with self-hosting help. We are a small team and we will try our best to help you.
+
+<Info>Community self-host option includes only community support. Checkout [commercial self host](/self-hosting-novu/commerical-self-host) for priority support.</Info>
+<Tip>Intercome is suitable if you're using Novu cloud or [commercial self-host](/self-hosting-novu/commerical-self-host) and want priority support. Kindly use Discord or Github for community self-hosting questions.</Tip>
+## Contributing
+
+All community members are of one of the following four types:
+
+1. Open source users
+2. Active users
+3. Power users
+
+### Become an active community member
+
+- Everyone who joins our community becomes an open-source user automatically.
+- You can become an active user by:
+    - Submitting >2 PRs
+    - Opening >2 issues
+    - Sending >5 messages on Github
+    - Commenting >3 times on Github
+- Active users are one step closer to becoming Power users.
+
+### Become a power user
+
+- All active community members are eligible to become a power user and can become so by having at least one of the following:
+    - Submitting 3 PRs
+    - Opening 3 Issues
+    - Sending 10 Discord messages
+    - Commenting >10 times on Github
+
+<Info>The activity period is 30 days for becoming an active/power user. So, you'll be upgraded/downgraded depending on your activity in the stipulated time.</Info>
+
+## Activities
+
+Are you passionate about notifications like we are? There are many different—and easy—ways to get involved.
+
+### Help in onboarding and answering questions.
+
+- Welcome new users into the community
+- Answer questions in the [community support channel]([link](https://discord.com/channels/895029566685462578/1019663407915483176)).
+- Offer input and opinions about various solutions you have tried.
+
+### Improve our documentation
+
+- Check out our[Github repo](https://github.com/novuhq/docs)
+- No documentation is perfect, and neither is ours
+- Help us improve our docs by
+    - Updating outdated examples
+    - Correcting typos and language for clarity
+    - Find and fix broken links, etc.
+
+
+### Help with the SDKs
+
+- Link to contributors guide(https://github.com/novuhq/novu/blob/next/CONTRIBUTING.md)
+- We have SDKs in various languages and frameworks, most written and maintained by community members.
+- You can either become a maintainer there or help the existing maintainers by bringing the SDK up to speed with the latest features present in the core product. (name and link all SDKs).
```

Comment:
(name and link all SDKs) - I believe you are meant to add them here.

Reply:
Yeah, my bad! 🙈🙈🙈